### PR TITLE
Flow project.json info into assembly info.

### PR DESCRIPTION
**Pull request by cesarbs**
#2715

### CODE REVIEW

**Comment by cesarbs** [comment]
@victorhurdugaci

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 DNX
 ===
 
-Travis:   [![Travis](https://travis-ci.org/aspnet/dnx.svg?branch=dev)](https://travis-ci.org/aspnet/dnx)
+Travis:   [![Travis](https://travis-ci.org/aspnet/dnx.svg?branch=cesarbs/dnx-assembly-info)](https://travis-ci.org/aspnet/dnx)
```
Comment (victorhurdugaci) [inline README.md]: Why do you update the branch?

Comment (cesarbs) [inline README.md]: To see the thingy going green in the repo page :)

Comment (moozzyk) [inline README.md]: volkswagen!

**Comment by victorhurdugaci** [comment]
No tests? (sad panda)

File: src/Microsoft.Dnx.Compilation.CSharp/RoslynCompiler.cs [hunk 2]
```diff
@@ -252,58 +252,44 @@ private CompilationModules GetCompileModules(CompilationTarget target)
...
 
-            var foundAssemblyFileVersion = false;
-            var foundAssemblyVersion = false;
-            var foundAssemblyInformationalVersion = false;
-
-            foreach (var assembly in assemblyAttributes)
+            foreach (var attribute in assemblyAttributes)
             {
-                string attributeName = assembly.AttributeClass.ToString();
+                string attributeName = attribute.AttributeClass.ToString();
```
Comment (JunTaoLuo) [inline src/Microsoft.Dnx.Compilation.CSharp/RoslynCompiler.cs]: nit: use var instead of string

File: README.md (hunk 1 above)
Comment (davidfowl) [inline README.md]: That's pretty weird :smile: You can just check travis itself.

File: src/Microsoft.Dnx.Compilation.CSharp/RoslynCompiler.cs [hunk 3]
```diff
@@ -252,58 +252,44 @@ private CompilationModules GetCompileModules(CompilationTarget target)
...
         }
 
-        private static CSharpCompilation ApplyVersionInfo(CSharpCompilation compilation, CompilationProjectContext project,
+        private static CSharpCompilation ApplyProjectInfo(CSharpCompilation compilation, CompilationProjectContext project,
             CSharpParseOptions parseOptions)
         {
-            const string assemblyFileVersionName = "System.Reflection.AssemblyFileVersionAttribute";
-            const string assemblyVersionName = "System.Reflection.AssemblyVersionAttribute";
-            const string assemblyInformationalVersion = "System.Reflection.AssemblyInformationalVersionAttribute";
+            var attributeNameToProjectValue = new Dictionary<string, string>(StringComparer.Ordinal)
```
Comment (davidfowl) [inline src/Microsoft.Dnx.Compilation.CSharp/RoslynCompiler.cs]: I don't see any checks for null or invalid values. What happens if the language is null or empty? same for copyright and description.

**Comment by davidfowl** [comment]
Add a test that verifies the assembly information is written after building. It should be part of the DnuPackTests

File: build.cmd [hunk 4]
```diff
@@ -4,8 +4,8 @@ cd %~dp0
 SETLOCAL
```
Comment (cesarbs) [inline build.cmd]: I have no idea what happened here.

**Comment by cesarbs** [comment]
Not flowing the "language" from project.json as AssemblyCulture into the assembly since it looks like that has already been handled by #668.

**Comment by cesarbs** [comment]
I will close this pr and open a new one, since this one got messed up with unrelated changes.

**Pull request closed without merging**